spec/acceptance/apt_keyring_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+PUPPETLABS_KEYRING_CHECK_COMMAND = 'gpg --import /etc/apt/keyrings/puppetlabs-keyring.gpg | gpg --list-keys | grep -F -A 1 \'pub   rsa4096 2019-04-08 [SC] [expires: 2025-04-06]\'' \
+'| grep \'D6811ED3ADEEB8441AF5AA8F4528B6CD9E61EF26\''
+
+describe 'apt::keyring' do
+  context 'when using default values and source specified explicitly' do
+    keyring_pp = <<-MANIFEST
+     apt::keyring { 'puppetlabs-keyring.gpg':
+  	  source => 'https://apt.puppetlabs.com/keyring.gpg',
```

Comment:
All tabs should be replaced with spaces. Configure your editor to use only spaces for indentation.